media-video/avidemux/avidemux-9999.ebuild

```diff
@@ -1,113 +1,121 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="5"
+EAPI="6"
 
 PLOCALES="ca cs de el es fr it ja pt_BR ru sr sr@latin tr"
 
-inherit cmake-utils eutils flag-o-matic l10n
-
-SLOT="2.6"
+inherit cmake-utils l10n
 
 DESCRIPTION="Video editor designed for simple cutting, filtering and encoding tasks"
 HOMEPAGE="http://fixounet.free.fr/${PN}"
 
 # Multiple licenses because of all the bundled stuff.
 LICENSE="GPL-1 GPL-2 MIT PSF-2 public-domain"
-IUSE="debug opengl nls qt4 sdl vaapi vdpau video_cards_fglrx xv"
-KEYWORDS="~amd64 ~x86"
+SLOT="2.6"
+IUSE="debug opengl nls nvenc qt4 sdl vaapi vdpau video_cards_fglrx xv"
 
 if [[ ${PV} == *9999* ]] ; then
-	KEYWORDS=""
-	EGIT_REPO_URI="git://gitorious.org/${PN}2-6/${PN}2-6.git https://git.gitorious.org/${PN}2-6/${PN}2-6.git"
+	MY_P="${P}"
+	EGIT_REPO_URI="https://github.com/mean00/avidemux2.git"
 
-	inherit git-2
+	inherit git-r3
 else
 	MY_P="${PN}_${PV}"
 	SRC_URI="mirror://sourceforge/${PN}/${PN}/${PV}/${MY_P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
 fi
 
 DEPEND="
-	~media-libs/avidemux-core-${PV}:${SLOT}[nls?,sdl?,vaapi?,vdpau?,video_cards_fglrx?,xv?]
+	~media-libs/avidemux-core-${PV}:${SLOT}[nls?,sdl?,vaapi?,vdpau?,video_cards_fglrx?,xv?,nvenc]
 	opengl? ( virtual/opengl:0 )
 	qt4? ( >=dev-qt/qtgui-4.8.3:4 )
 	vaapi? ( x11-libs/libva:0 )
+	nvenc? ( media-video/nvidia_video_sdk )
 	video_cards_fglrx? (
 		|| ( >=x11-drivers/ati-drivers-14.12-r3
 			x11-libs/xvba-video:0 )
 		)"
-RDEPEND="$DEPEND"
+RDEPEND="
+	$DEPEND
+	nls? ( virtual/libintl:0 )
+"
 PDEPEND="~media-libs/avidemux-plugins-${PV}:${SLOT}[opengl?,qt4?]"
 
 S="${WORKDIR}/${MY_P}"
 
 src_prepare() {
-	cmake-utils_src_prepare
+	eapply_user
+
+	processes="buildCli:avidemux/cli"
+	use qt4 && processes+=" buildQt4:avidemux/qt4"
+
+	for process in ${processes} ; do
+		local build="${WORKDIR}/${P}_build/${process%%:*}"
+
+		mkdir -p "${build}" || die "Can't create build folder."
+		cd "${build}" || die "Can't enter build folder."
```

Comment:
Actually, it seems the BUILD_DIR is created in cmake-utils_src_configure if it doesn't exist, so those statements should be unnecessary.